docs/services/gcp-services-guide.md

```diff
@@ -0,0 +1,536 @@
+  # to export it under. This is the name of the secret
+  # in the crossplane cluster, and it's scoped to this claim's namespace.
+  writeConnectionSecretToRef:
+    name: wordpressmysql
+EOF
+
+kubectl apply -f mysql-claim.yaml
+```
+
+What we are looking for is for `STATUS` value to become `Bound` which indicates
+the managed resource was successfully provisioned and is ready for consumption.
+You can see when claim is bound using the following:
+
+*Command*
+```bash
+kubectl get mysqlinstances -n app-project1-dev
+```
+
+*Output*
+```bash
+NAME          STATUS   CLASS            VERSION   AGE
+mysql-claim   Bound    mysql-standard   5.6       11m
+```
+
+If the `STATUS` is blank, we are still waiting for the claim to become bound.
+You can observe resource creation progression using the following:
+
+*Command*
+```bash
+kubectl describe mysqlinstance mysql-claim -n app-project1-dev
+```
+
+*Output*
+```
+Name:         mysql-claim
+Namespace:    app-project1-dev
+Labels:       <none>
+Annotations:  kubectl.kubernetes.io/last-applied-configuration:
+                {"apiVersion":"database.crossplane.io/v1alpha1","kind":"MySQLInstance","metadata":{"annotations":{},"name":"mysql-claim","namespace":"team..."}}
+API Version:  database.crossplane.io/v1alpha1
+Kind:         MySQLInstance
+Metadata:
+  Creation Timestamp:  2019-09-16T13:46:42Z
+  Finalizers:
+    finalizer.resourceclaim.crossplane.io
+  Generation:        2
+  Resource Version:  4256
+  Self Link:         /apis/database.crossplane.io/v1alpha1/namespaces/app-project1-dev/mysqlinstances/mysql-claim
+  UID:               6a7fe064-d888-11e9-ab90-42b6bb22213a
+Spec:
+  Class Ref:
+    Name:          mysql-standard
+  Engine Version:  5.6
+  Resource Ref:
+    API Version:  database.gcp.crossplane.io/v1alpha2
+    Kind:         CloudsqlInstance
+    Name:         mysqlinstance-6a7fe064-d888-11e9-ab90-42b6bb22213a
+    Namespace:    gcp-infra-dev
+  Write Connection Secret To Ref:
+    Name:  wordpressmysql
+Status:
+  Conditions:
+    Last Transition Time:  2019-09-16T13:46:42Z
+    Reason:                Managed claim is waiting for managed resource to become bindable
+    Status:                False
+    Type:                  Ready
+    Last Transition Time:  2019-09-16T13:46:42Z
+    Reason:                Successfully reconciled managed resource
+    Status:                True
+    Type:                  Synced
+Events:                    <none>
+```
+
+*Note: You must wait until the claim becomes bound before continuing with this
+guide. It could take a few minutes for GCP to complete CloudSQL creation.*
+
+#### Connecting MySQL Instance and GKE Cluster
+Before we install Wordpress, we need establish connectivity between our MySQL
+database and our GKE cluster. We can do this by creating a [Private Service 
+Connection](https://cloud.google.com/vpc/docs/configure-private-services-access).
+
+You can create it following the link above or use Crossplane to do it:
```

Comment:
`You can create it by following the instructions at the link above, or you could use Crossplane to do it:` would be more grammatically correct